activejob/lib/active_job/logging.rb

```diff
@@ -68,10 +68,13 @@ def enqueue_at(event)
         end
 
         def perform_start(event)
+          job = event.payload[:job]
           info do
-            job = event.payload[:job]
             "Performing #{job.class.name} (Job ID: #{job.job_id}) from #{queue_name(event)}" + args_info(job)
           end
+          info do
+            "#{job.class.name} (Job ID: #{job.job_id}) waited in the queue for #{job.enqueded_for}ms"
+          end
```

Comment:
Can we make the wait time part of the other log message?  Then we don't need to have two `info` blocks.

I believe the `job = event...` line was moved inside the info block for performance (so it doesn't get executed unless the logger is in info mode).